src/pages/ProductDetail/BreakDownModal/BreakDownModal.js

```diff
@@ -0,0 +1,397 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function BreakDownModal({ closeModal }) {
+  const [breakDown, setBreakDown] = useState(CONTRACT_DATA);
+  const [tradeBtn, setTradeBtn] = useState([true, false, false]);
+  const [tradeHeader, setTradeHeader] = useState(SIGN_HEADER);
```

Comment:
```suggestion
  const [currentId, setCurrentId] = useState(1);
```

해당 부분도 선택된 아이디 값에 따라서 UI를 관리해 줄 수 있습니다.
CONTRACT_DATA, SIGN_HEADER는 결국 상수데이터기 때문에
state로 사용하기 부적절한 부분입니다!